src/Illuminate/Http/UploadedFile.php

```diff
@@ -16,23 +16,25 @@ class UploadedFile extends SymfonyUploadedFile
      *
      * @param  string  $path
      * @param  string|null  $disk
+     * @param  string|bool|null  $mimeType
      * @return string|false
      */
-    public function store($path, $disk = null)
+    public function store($path, $disk = null, $mimeType = null)
     {
-        return $this->storeAs($path, $this->hashName(), $disk);
+        return $this->storeAs($path, $this->hashName(), $disk, $mimeType);
```

Comment:
Following @caiquecastro comment...
There should be a ``` null ``` here before ``` $mimeType ```, currently you're passing the ``` $mimeType ``` as the ``` $visibility ``` parameter.